src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.EolTargetFrameworks.props

```diff
@@ -0,0 +1,25 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.EolTargetFrameworks.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Default the check to true, but allow developers to disable the warning. -->
+    <CheckEolTargetFramework Condition="'$(CheckEolTargetFramework)' == ''">true</CheckEolTargetFramework>
+  </PropertyGroup>
+
+  <!--
+    .NET Core framework versions that are currently considered to be end-of-life (EOL). These versions will not be targeted to
+    receive servicing updates and security fixes.
+  -->
+  <ItemGroup>
+    <_EolNetCoreTargetFrameworkVersionsWithoutV Include="1.0;1.1;2.0;2.2;3.0" />
```

Comment:
I'd just rename this to _EolNetCoreTargetFrameworkVersions

Reply:
fixed in the latest PR